CMakeLists.txt

```diff
@@ -285,6 +285,8 @@ if(OCTREE_DEBUG)
   add_definitions(-DOCTREE_DEBUG)
 endif()
 
+include(FindThreads)
```

Comment:
why did you add this check? `std::thread` is a c++11 feature available on all platforms. While this check look for posix threads.

Reply:
AFAIR without it linker throwed error.